packages/react-components/src/style/output-components-style.css

```diff
@@ -9,6 +9,7 @@
     background-color: var(--theia-layout-color4);
     display: grid;
     grid-template-rows: 25px 1fr;
+    cursor: url('cross-arrows.png'), auto ;
```

Comment:
In trace-context-component, the  class used for dragging widget is the `title-bar-label`, so this should be the class whose style to update. The widget-handle is bigger than the title-bar-label, so sometimes, the cursor is the cross-arrows but it is not draggable.

Reply:
Thanks for your feedback, I didn't know that there was a light theme, so I choosed the icon based on the dark one. I will change it.

I took the icon from: 
 https://favpng.com/png_view/arrow-arrow-cross-cursor-symbol-png/zycH9794

I changed the color since it was dark and resized it to 20 pixels because it was too large.

Concerning your second comment, I will make the changes.

Reply:
@ebugden , here is the video

Do you have any suggestion for the icon I should use ?


https://user-images.githubusercontent.com/46821405/104362505-e158cc80-54e1-11eb-986d-de9ee66c4c5e.mp4

Reply:
What license is the icon originally? I can't find anything on their site. :(

Reply:
I can't find it either :/
Since there is no cross-arrow icon (from what I saw) in the CSS libraries, I took an image from the internet just to start.

Reply:
![image](https://user-images.githubusercontent.com/419535/104364113-23830d80-54e4-11eb-9fa1-ea7aa30e9d2d.png)
hand drawn pixel by pixel licensed under epl right here ^ if you guys need it.

Reply:
Also if you want a wider arrow, here you go. 
![image](https://user-images.githubusercontent.com/419535/104364280-67761280-54e4-11eb-9e03-5be7d07e3c1b.png)

Reply:
Looks like you already have an icon, but one suggestion is to use the native browser `move` cursor:
```css
cursor: move;
```
That way you don't have to worry about licensing and you won't need to add another image asset. You also get accessibility contrast for free. Downside is that you don't get ultimate control of what the cross-arrow icon looks like. On Windows/Chrome, this is roughly what it looks like:

![move-cursor-resized](https://user-images.githubusercontent.com/14880569/104475241-7b715100-5584-11eb-953f-dbadc5f8fcf2.png)

Just a thought! Feel free to ignore if you are happy with your current icon/approach 😃

Reply:
I agree with @danieltomasku  that using `cursor: move` solves many issues with hand drawn icons (whose edges are not as clear in light mode). 

![moveLightMode](https://user-images.githubusercontent.com/4398897/104774826-55df7580-5745-11eb-948f-33c703750059.png)


But then, the icon is not the same everywhere. In Linux/Firefox, it is a little fist dragging the window: ![moveIcon](https://user-images.githubusercontent.com/4398897/104775232-03528900-5746-11eb-81ac-ca0842a35a27.png) Just how acceptable is that?

Reply:
I still think we should take this option. I think it's the obvious good first choice and then we can adjust if users don't like it.